packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -639,13 +646,37 @@ class _UiKitViewState extends State<UiKitView> {
       layoutDirection: _layoutDirection!,
       creationParams: widget.creationParams,
       creationParamsCodec: widget.creationParamsCodec,
+      onFocus: () {
+        _focusNode!.requestFocus();
+      }
     );
     if (!mounted) {
       controller.dispose();
       return;
     }
     widget.onPlatformViewCreated?.call(id);
-    setState(() { _controller = controller; });
+    setState(() {
+      _controller = controller;
+      _id = id;
```

Comment:
Per previous comment, does the `focusNode` `debugLabel` need to be updated when `_id` changes?